pkg/dns/ibm/dns.go

```diff
@@ -139,9 +143,18 @@ func (p *Provider) createOrUpdateDNSRecord(record *iov1.DNSRecord, zone configv1
 	if !ok {
 		return fmt.Errorf("createOrUpdateDNSRecord: unknown zone: %v", zone.ID)
 	}
+
+	// TTL must be between 120 and 2,147,483,647 seconds, or 1 for Automatic.
+	if (record.Spec.RecordTTL > 1) && (record.Spec.RecordTTL < 120) {
```

Comment:
Wouldn't it be better to convert to the default, in case they didn't mean for 0 to be converted to 1?  If not, just change the documentation in the comments to explain why you chose to allow 0 to convert to 1 rather than the default.

```suggestion
	if record.Spec.RecordTLL !=1  && record.Spec.RecordTTL < 120 {
```

Reply:
Makes sense. Changed it to convert 0 to default.